fix-version.js

```diff
@@ -0,0 +1,6 @@
+const path = require('path');
+const packageJsonPath = path.join(process.cwd(), './package.json');
+const packageJson = require(packageJsonPath);
+packageJson.version = packageJson.version.replace('2.2.1', '4.0.0');
```

Comment:
What this is for? Looks like hack

Reply:
Somehow, lerna generates `canary` version as `2.2.1-alpha.XXXX` but currently we have 3.x version as stable and `4.x` as the future version. So I added this hacky script until we release the stable version of 4.x.